src/test/crt_echo.h

```diff
@@ -183,6 +183,14 @@ parse_options(int argc, char *argv[])
 	}
 }
 
+int my_func1(crt_context_t ctx, crt_rpc_t *rpc, void (rpc_handler)(void *), void *arg) {
```

Comment:
(style)  line over 80 characters
(style)  open brace '{' following function declarations go on the next line